core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/AwsV4aAuthScheme.java

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.http.auth;
+package software.amazon.awssdk.http.auth.aws;
```

Comment:
Should this class return the new signer on line 66, or is that part of a different PR?